tests/unit/browser/webkit/test_downloads.py

```diff
@@ -30,6 +30,30 @@ def test_download_model(qapp, qtmodeltester, config_stub, cookiejar_and_cache):
     qtmodeltester.check(model)
 
 
+@pytest.mark.parametrize('url, title, out', [
+    ('https://qutebrowser.org/img/cheatsheet-big.png',
+     'cheatsheet-big.png (3342×2060)',
+     None),
+    ('http://qutebrowser.org/INSTALL.html',
+     'Installing qutebrowser | qutebrowser',
+     'Installing qutebrowser _ qutebrowser.html'),
+    ('http://qutebrowser.org/INSTALL.html.html',
+     'Installing qutebrowser | qutebrowser',
+     'Installing qutebrowser _ qutebrowser.html'),
```

Comment:
I don't understand this test - I think to resemble the behavior with the double extension, this should have a `INSTALL.html` filename (not `.html.html`) and a title which ends with `.html`, no?